pyproject.toml

```diff
@@ -1,3 +1,66 @@
+[tool.poetry]
+name = "vcrpy"
+version = "6.0.1"
+description = "Automatically mock your HTTP interactions to simplify and speed up testing"
+authors = ["Kevin McCarthy <me@kevinmccarthy.org>"]
+readme = "README.rst"
+packages = [
+    { include = 'vcr' },
+]
+license = "MIT"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Only",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Internet :: WWW/HTTP",
+    "License :: OSI Approved :: MIT License"
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8"
+pyyaml = "^6.0.1"
+wrapt = "^1.16.0"
+yarl = "^1.9.4"
+urllib3 = [
+    { version = "<2", markers = "python_version < '3.10'" },
+    { version = "<2", markers = "platform_python_implementation == 'PyPy'" },
+    { version = ">=2.2.2", markers = "python_version >= '3.10' and platform_python_implementation != 'PyPy'" }
+]
```

Comment:
after thought. This is incorrect. This package has no dependencies as it is, itself a dev dependency.